src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -16,8 +16,8 @@ The following options can be configured on the root level of your browser-based
 
 | Key                      | Type   | Default | Description                                                                                                                                                                                                                                                       |
 | ------------------------ | ------ | ------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| replaysSessionSampleRate | number | `0`   | The sample rate for replays that begin recording immediately and last the entirety of the user's session. `1.0` collects all replays, and `0` collects none.                                                                                                      |
-| replaysOnErrorSampleRate | number | `0`   | The sample rate for replays that are recorded when an error happens. This type of replay will record up to a minute of events prior to the error and continue recording until the session ends. `1.0` captures all sessions with an error, and `0` captures none. |
+| replaysSessionSampleRate | number | `0`     | The sample rate for replays that begin recording immediately and last the entirety of the user's session. `1.0` collects all replays, and `0` collects none.                                                                                                      |
```

Comment:
Just some auto-formatting (TIL that prettier auto-formats markdown tables - neat :D)